inventory/rh-sso/group_vars/rh-sso-hosts.yml

```diff
@@ -6,22 +6,26 @@ ansible_become: True
 list_of_packages_to_install:
   - "@rh-sso7"
 
+sso_protocol: https
```

Comment:
Make sure to set this in the `defaults/main.yml` as well

Reply:
... and preferably align with the existing naming - i.e.: `rh_sso....`

Reply:
Fixed.